about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>about</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="my-nav">
+        <div class="top-bar">
+          <a href=""
+            ><img class="insta" src="./images/instagram-30.png" alt=""
+          /></a>
+          <p>Cindy Shin</p>
+        </div>
+        <div class="my-logo">
+          <a href="index.html"> <img src="./images/cblogo.png" alt="link" /></a>
+        </div>
+        <div class="hamburger" id="hamburger">
+          <img src="./images/more.png" alt="hamburger icon" />
+        </div>
+        <div class="nav-bar">
+          <ul>
+            <li>
+              <a href="about.html"><p>About</p></a>
+            </li>
+            <li>
+              <a href="#programme" id="programpage"><p>Program</p></a>
+            </li>
+            <li>
+              <a href="#sponserr" id="sponsorpage"><p>Sponsor</p></a>
+            </li>
+            <li>
+              <a href="" id="newspage"><p>News</p></a>
+            </li>
```

Comment:
- [ ] Please note that on the` About` desktop page there is no link to navigate to the home page Kindly fix that😉

Reply:
Fixed!! 👍🏽